shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -227,6 +257,7 @@ void AccessibilityBridge::AddSemanticsNodeUpdate(
         .set_transform(GetNodeTransform(flutter_node))
         .set_attributes(GetNodeAttributes(flutter_node, &this_node_size))
         .set_states(GetNodeStates(flutter_node, &this_node_size))
+        .set_actions(GetNodeActions(flutter_node, &this_node_size))
```

Comment:
I am not familiar with fuchsia code. Do you know where are these APIs defined?

Reply:
Are you talking about the fuchsia node API? https://source.corp.google.com/piper///depot/google3/third_party/unsupported_toolchains/fuchsia/sdk/fidl/fuchsia_accessibility_semantics/node.fidl?q=fuchsia%20node%20fidl&ct=os&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental

Reply:
ah I see, thanks for the link